HousingSearchApi/V1/Domain/Person.cs

```diff
@@ -0,0 +1,103 @@
+using System.Collections.Generic;
+using HousingSearchApi.V1.Domain.ES;
+
+namespace HousingSearchApi.V1.Domain
+{
+    public class Person
+    {
+        public static Person Create(QueryablePerson person)
+        {
+            return new Person
+            {
+                Id = person.Id,
+                Title = person.Title,
+                Firstname = person.Firstname,
+                MiddleName = person.MiddleName,
+                Surname = person.Surname,
+                PreferredFirstname = person.PreferredFirstname,
+                PreferredSurname = person.PreferredSurname,
+                Ethinicity = person.Ethinicity,
+                Nationality = person.Nationality,
+                PlaceOfBirth = person.PlaceOfBirth,
+                DateOfBirth = person.DateOfBirth,
+                Gender = person.Gender,
+                Identification = Create(person.Identification ?? new List<ES.Identification>()),
+                PersonTypes = person.PersonTypes,
+                IsPersonCautionaryAlert = person.IsPersonCautionaryAlert,
+                IsTenureCautionaryAlert = person.IsTenureCautionaryAlert,
+                Tenures = Create(person.Tenures ?? new List<ES.Tenures>())
+            };
+        }
+
+        private static List<Identification> Create(List<ES.Identification> identifications)
+        {
+            var identList = new List<Identification>();
+
+            foreach (ES.Identification identification in identifications)
+            {
+                identList.Add(new Identification
+                {
+                    IdentificationType = identification.IdentificationType,
+                    LinkToDocument = identification.LinkToDocument,
+                    OriginalDocumentSeen = identification.OriginalDocumentSeen,
+                    Value = identification.Value
+                });
+            }
+
+            return identList;
+        }
+
```

Comment:
Should this not be in the factories folder (or at least separate from this file)